charts/kube-state/values.yaml

```
@@ -27,4 +27,5 @@ ports:
 
 replicas: 1
 
-extraArgs: []
+extraArgs:
+- --metric-labels-allowlist=namespaces=[*]
```

Just to make this a bit more clear and safe

```suggestion
- "--metric-labels-allowlist=namespaces=[*]"
```